src/crypto/ms_oapxbc.rs

```diff
@@ -29,72 +24,56 @@ const CTX_NONCE_LEN: usize = 32;
 pub enum MsOapxbcSessionKey {
     /// An AES-256-GCM/CBC + HS256 session key
     A256GCM {
-        /// The aes key for this session
-        aes_key: [u8; A256_KEY_LEN],
+        /// The encrypted session key which can be loaded as required for individual operations
+        loadable_session_key: LoadableMsOapxbcSessionKey,
```

Comment:
This could instead be:

```rust
pub enum MsOapxbcSessionKey {
    /// An AES-256-GCM/CBC + HS256 session key
    A256GCM {
        #[cfg(not(feature = "hsm-crypto"))]
        aes_key: [u8; A256_KEY_LEN],
        #[cfg(feature = "hsm-crypto")]
        loadable_session_key: LoadableMsOapxbcSessionKey,
    },
]
```